extras/devices/LVFS/tests/fwupd_data.py

```diff
@@ -0,0 +1,387 @@
+""" fwupdmgr json outputs """
+
+
+get_results = """{
```

Comment:
The names of the "constants" in this file sound more like functions. Can you please rename it to something like "XXX_RESPONSE_DATA" (replacing XXX of course)

Reply:
Done.